roles/ceph-common/tasks/installs/install_redhat_packages.yml

```diff
@@ -84,15 +84,24 @@
     - nfs_obj_gw
     - ceph_rhcs_version == "2"
```

Comment:
@leseb @ktdreyer  are there supposed to be separate packages for nfs-ganesha for ceph_rhcs_version 2? I was tempted to remove that task, but I wanted to run it by you two first.

Reply:
RHCS 2 has a single older "nfs-ganesha-fsal" .deb.
RHCS 3 has the split .debs, "nfs-ganesha-rgw" and "nfs-ganesha-ceph".